mx.py

```diff
@@ -2222,6 +2222,9 @@ def resolveDeps(self):
         if licenseId:
             self.theLicense = get_license(licenseId, context=self)
 
+    def substVars(self, text):
+        return text.format(**vars(self))
```

Comment:
This means the substitute values may not be the values of the suite.py attributes named in `text`. Couldn't this cause confusion?